robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -285,7 +290,7 @@ def fill_gift_entry_form(self,**kwargs):
             print(f"type is {type}")
             if 'autocomplete' in type :
                 self.salesforce._populate_field(locator,value)
-                value_locator=npsp_lex_locators["gift_entry"]["id"].format(value)
+                value_locator=npsp_lex_locators["gift_entry"]["id"].format("Select "+value)
```

Comment:
Can you not add the word "Select" in the locator itself and just parameterize the value ? So that you can use it as:
`value_locator=npsp_lex_locators["gift_entry"]["id"].format(value)` ??
Or are you repurposing this locator elsewhere too ?

Reply:
yes this particular locator is being reused in other places.